packages/core/src/shared/extensionUtilities.ts

```diff
@@ -321,17 +321,19 @@ export function getComputeRegion(): string | undefined {
 }
 
 /**
- * Provides an {@link vscode.Event} that is triggered when the user interacts with the extension.
- * This will help to be notified of user activity.
+ * Provides a generic {@link vscode.Event} representing "user activity".
  */
-export class ExtensionUserActivity implements vscode.Disposable {
+export class UserActivity implements vscode.Disposable {
```

Comment:
Renamed. This event is intended to track any kind of "user active" signal, not only "extension" activity.